src/main/java/it/gov/pagopa/paymentupdater/model/Message.java

```diff
@@ -16,7 +18,7 @@ public class Message {
 	protected String senderServiceId;
 	protected String senderUserId="undefined";
 	protected int timeToLiveSeconds;
-	private Long dueDate;
+	private LocalDateTime dueDate;
```

Comment:
We must check how this date is serialized to avro

Reply:
The message  is deserialized in the automatically generated **'message'** (lowercase m) class. The conversion to LocalDateTime is done later when we go to save the information.